infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -199,6 +203,23 @@ public static void main(String[] args) {
 
         log.debug("Started with arguments: " + opts);
 
+        // Bind to all interfaces only if no address or interface specified by the user.
+        final boolean bindToAllInterfaces;
+        // Fetch the address if given a network interface.
+        if (opts.get("--network-interface") != null) {
+            opts.put("--address",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "--address" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)